src/Microsoft.AspNetCore.Mvc.Formatters.Json/DependencyInjection/MvcJsonMvcCoreBuilderExtensions.cs

```diff
@@ -47,6 +47,24 @@ public static IMvcCoreBuilder AddJsonFormatters(this IMvcCoreBuilder builder)
             return builder;
         }
 
+        public static IMvcCoreBuilder AddJsonOptions(
```

Comment:
It's a public extension method for a public type - no harm adding docs for this.